src/components/common/CryptoExchangeWalletListRow.js

```diff
@@ -0,0 +1,153 @@
+// @flow
+import { bns } from 'biggystring'
+import type { EdgeDenomination } from 'edge-core-js'
+import _ from 'lodash'
+import React, { Component } from 'react'
+import { Image, Text, TouchableHighlight, View } from 'react-native'
+
+import { intl } from '../../locales/intl'
+import type { State } from '../../modules/ReduxTypes'
+import * as SETTINGS_SELECTORS from '../../modules/Settings/selectors.js'
+import FormattedText from '../../modules/UI/components/FormattedText/index'
+import { calculateSettingsFiatBalance } from '../../modules/UI/selectors.js'
+import { CryptoExchangeWalletListRowStyle as styles } from '../../styles/indexStyles'
+import type { GuiWallet } from '../../types'
+import { decimalOrZero, getCurrencyAccountFiatBalanceFromWallet, getFiatSymbol, truncateDecimals } from '../../util/utils.js'
+import { CryptoExchangeWalletListTokenRow } from './CryptoExchangeWalletListTokenRow.js'
+
+type Props = {
+  wallet: GuiWallet,
+  onPress(GuiWallet): void,
+  onTokenPress(Object): void,
+  state: State
+}
+type LocalState = {
+  fiatBalance: string,
+  fiatSymbol: string,
+  isWalletFiatBalanceVisible: boolean,
+  cryptoBalance: string,
+  cryptoSymbol: string,
+  enabledNativeBalances: Object
+}
+
+const DIVIDE_PRECISION = 18
+
+class CryptoExchangeWalletListRow extends Component<Props, LocalState> {
+  constructor (props: Props) {
+    super(props)
+    this.state = {
+      fiatBalance: '',
+      isWalletFiatBalanceVisible: true,
+      cryptoBalance: '',
+      cryptoSymbol: '',
+      enabledNativeBalances: {},
+      fiatSymbol: ''
+    }
+  }
+  componentDidMount () {
+    const settings = this.props.state.ui.settings
+    const denomination: EdgeDenomination = SETTINGS_SELECTORS.getDisplayDenomination(this.props.state, this.props.wallet.currencyCode)
```

Comment:
Caveat: this will not live update as things change. Seems appropriate for the use-case, just wanted to make a note for posterity.